tests/failpoint/db_failpoint_test.go

```diff
@@ -1,6 +1,8 @@
 package failpoint
 
 import (
+	"fmt"
+	"go.etcd.io/bbolt/internal/btesting"
```

Comment:
minor issue: please move it into third group.

Reply:
Good catch. Thank you @fuweid 

We may want to enhance the workflow to detect such error automatically.

Reply:
raised https://github.com/etcd-io/bbolt/issues/435